task-list/app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<header>
+  🔅🔅🔆 Task Organizer 🔆🔅🔅
+</header>
+<nav>
+  <%= link_to "Task List", "/" %>
+</nav>
+<h1>Task Details</h1>
+  <p><b>Name:</b> <%= @task.name %></p>
```

Comment:
Remember our discussion about semantic HTML, using `strong` is better than using `b` because it provides more context about _why_ your page is structured the way it is